modules/integration_azure-datafactory/common-modules.tf

```diff
@@ -0,0 +1 @@
+../../common/module/modules.tf
```

Comment:
The common modules are not so common and this file must be overriden for Azure detectors.
See https://github.com/claranet/terraform-signalfx-detectors/pull/232/files#diff-bb6d827b32ed7a279afebaba971ac107164c91173cdf2a40cae959f6163d03a4